src/Codeception/Lib/Driver/Db.php

```diff
@@ -32,10 +32,36 @@ class Db
      * @var array
      */
     protected $primaryKeys = [];
+    /**
+     * @var string|null
+     */
+    protected $pdo_class;
+
+    /**
+     * @param string|null $pdo_class
+     * @return string
+     */
+    private static function pdoClass($pdo_class){
+        if (!$pdo_class){
+            // If empty or null we use regular PDO
+            return \PDO::class;
+        }
+
+        if (!class_exists($pdo_class)){
+            throw new ModuleException(
```

Comment:
Initially this returned the fallback value of `\PDO::class` but actually throwing an exception is more helpful to the user - they can resolve by correcting the class name _or_ by simply removing the config item.